tools/edta/edta.xml

```diff
@@ -0,0 +1,472 @@
+    ## Genome to TE annotations 
+    #if $function_select.function == 'genome':
+        ln -s '$function_select.genome' ./input.fa &&
+        perl /EDTA/EDTA.pl 
+        --genome input.fa
+        --species '$function_select.species'
+        --step '$function_select.step'
+        #if $function_select.cds:
+            --cds '$function_select.cds'
+        #end if 
+        #if $function_select.curatedlib:
+            --curatedlib '$function_select.curatedlib'
+        #end if
+        #if $function_select.sensitive:
+            --sensitive 1
+        #end if
+        #if $function_select.mutation_rate:
+            --u "$function_select.mutation_rate"
+        #end if 
+        #if $function_select.anno_select.anno == 'yes':
+            --anno 1
+            #if $function_select.anno_select.evaluate:
+                --evaluate 1 
+            #end if
+            #if $function_select.anno_select.exclude:
+                --exclude '$function_select.anno_select.exclude'
+            #end if
+            &&  mv ./input.fa.mod.EDTA.anno/*.sum .
+        #end if 
+    ## Find elements of a paticular TE type
+    #else if $function_select.function == 'te':
+        ln -s '$function_select.genome' ./input.fa &&
+        perl /EDTA/EDTA_raw.pl
+        --genome input.fa
+        --species '$function_select.species'
+        --type '$function_select.te_type'
+        && mv ./input.fa.mod.EDTA.raw/input.fa.mod* .
+    ## pan-EDTA
+    #else if $function_select.function =='pan-edta':
+        #set $cds_list = []
+        #for $i, $s in enumerate($function_select.te_library)
+            ln -s '${s.genome}' ./${i}_input.fa &&
+            perl /EDTA/EDTA.pl 
+            --genome ${i}_input.fa 
+            --species '${s.species}'
+            --anno 1
+            #if $s.cds:
+                --cds '${s.cds}'
+            #end if 
+            #if $s.curatedlib:
+                --curatedlib '${s.curatedlib}'
+            #end if
+            #if $s.sensitive:
+                --sensitive 1
+            #end if
+            #if $s.mutation_rate:
+                --u "${s.mutation_rate}"
+            #end if
+            #if $s.evaluate:
+                --evaluate 1 
+            #end if
+            #if $s.exclude:
+                --exclude '${s.exclude}'
+            #end if
+            #silent$cds_list.append(str($i) + '_input.fa')
+            &&  mv ./${i}_input.fa.mod.EDTA.anno/*.mod.out .
+            &&      
+        #end for
+        #for $i in $cds_list
+            bash $__tool_directory__/filter_out_single_copies.sh $i  &&
+        #end for
+        #if $function_select.known_te:
+            bash $__tool_directory__/make_pan_library.sh '${known_te}' 
+        #else:
+            bash $__tool_directory__/make_pan_library.sh
+        #end if 
+        #for $i in $cds_list
+            bash $__tool_directory__/annotate_pan_genome.sh $i  &&
+        #end for
+        #for $i, $s in enumerate($function_select.te_library)
```

Comment:
not sure why those symlinks are needed, but if the command ends with "&&" this will result in an invalid command